src/interfaces/coral_web/src/components/EditEnvVariablesButton.tsx

```diff
@@ -92,7 +104,11 @@ const EditEnvVariablesModal: React.FC<{ onClose: () => void }> = ({ onClose }) =
 
     try {
       setIsSubmitting(true);
-      await updateDeploymentEnvVariables({ name: deployment, env_vars: envVariables });
```

Comment:
does this mean we don't need this endpoint? could we delete it :o

Reply:
We will most likely need it in the future.